src/accelerate/commands/launch.py

```diff
@@ -343,14 +343,14 @@ def launch_command_parser(subparsers=None):
     )
     deepspeed_args.add_argument(
         "--zero3_init_flag",
-        default=None,
+        default="true",
         type=str,
         help="Decides Whether (true|false) to enable `deepspeed.zero.Init` for constructing massive models. "
         "Only applicable with DeepSpeed ZeRO Stage-3.",
```

Comment:
may be add to `help` that `ACCELERATE_DEEPSPEED_ZERO3_INIT` env var can override it?

Reply:
Hello, `ACCELERATE_DEEPSPEED_ZERO3_INIT` is set using `zero3_init_flag` arg, so I don't think explicit mention is required as it is only used internally.

Reply:
ah, understood, thank you!